.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  root: true,
+  env: {
+    es2017: true,
```

Comment:
We could use `es2020` and bump it for the `tsconfig.build.json`.

Reply:
Let's stay conservative here, I think es2020 will break on Node 10 and we still want to support it in the near future.